Author: Robert James Clay <jame@rocasa.us>
Description: Set working directory in both the required ledgersmb_starman.service
 file as well as the three other optional conf/systemd service files. Include
 setting the LSMB_CONFIG_FILE variable in the environment. Also update the service
 file Descriptions to include the "1.7" version number.
Origin: vendor
Last-Update: 2019-07-04
--- a/doc/conf/systemd/ledgersmb-development_plackup.service
+++ b/doc/conf/systemd/ledgersmb-development_plackup.service
@@ -3,14 +3,14 @@
 # service start ledgersmb-development_plackup
 
 [Unit]
-Description=LedgerSMB Plack Development
+Description=LedgerSMB 1.7 Plack Development
 After=network.target
 
 [Service]
-WorkingDirectory=WORKING_DIR
+WorkingDirectory=/usr/share/ledgersmb-1.7
 
 #If you do not want a log file, comment this and remove the access_log option below
-Environment=LOG=/var/log/ledgersmb/ledgersmb-development_plackup.log
+Environment=LOG=/var/log/ledgersmb-1.7/development_plackup.log
 
 # In case you installed dependencies into a 'local::lib'
 # make sure you set the PERL5LIB environment variable
@@ -19,8 +19,8 @@
 # Be sure to set a user and group below
 # which don't have write access to the directories
 # holding the LedgerSMB sources
-User=ledgersmb
-Group=ledgersmb
+User=ledgersmb-1_7
+Group=ledgersmb-1_7
 
 # Use --Reload to monitor files changes and restart the server
 # The list below is exhaustive, change it to suit your needs
--- a/doc/conf/systemd/ledgersmb_plack-fcgi.service
+++ b/doc/conf/systemd/ledgersmb_plack-fcgi.service
@@ -3,11 +3,11 @@
 # service start ledgersmb_plack-fcgi
 
 [Unit]
-Description=LedgerSMB Plack/FCGI
+Description=LedgerSMB 1.7 Plack/FCGI
 After=network.target
 
 [Service]
-WorkingDirectory=WORKING_DIR
+WorkingDirectory=/usr/share/ledgersmb-1.7
 
 # In case you installed dependencies into a 'local::lib'
 # make sure you set the PERL5LIB environment variable
@@ -18,8 +18,8 @@
 # Be sure to set a user and group below
 # which don't have write access to the directories
 # holding the LedgerSMB sources
-User=ledgersmb
-Group=ledgersmb
+User=ledgersmb-1_7
+Group=ledgersmb-1_7
 ExecStart=/usr/bin/plackup                     \
     -s FCGI                                    \
     --listen localhost:3244                    \
--- a/doc/conf/systemd/ledgersmb_plackup.service
+++ b/doc/conf/systemd/ledgersmb_plackup.service
@@ -3,14 +3,14 @@
 # service start ledgersmb_plackup
 
 [Unit]
-Description=LedgerSMB Plack
+Description=LedgerSMB 1.7 Plack
 After=network.target
 
 [Service]
-WorkingDirectory=WORKING_DIR
+WorkingDirectory=/usr/share/ledgersmb-1.7
 
 #If you do not want a log file, comment this and remove the access_log option below
-Environment=LOG=/var/log/ledgersmb/ledgersmb_plackup.log
+Environment=LOG=/var/log/ledgersmb/ledgersmb-1-7_plackup.log
 
 # In case you installed dependencies into a 'local::lib'
 # make sure you set the PERL5LIB environment variable
@@ -19,8 +19,8 @@
 # Be sure to set a user and group below
 # which don't have write access to the directories
 # holding the LedgerSMB sources
-User=ledgersmb
-Group=ledgersmb
+User=ledgersmb-1_7
+Group=ledgersmb-1_7
 
 # Use -R to monitor files changes and restart the server
 ExecStart=/usr/bin/plackup   \
--- a/doc/conf/systemd/ledgersmb_starman.service
+++ b/doc/conf/systemd/ledgersmb_starman.service
@@ -3,22 +3,23 @@
 # service start ledgersmb_starman
 
 [Unit]
-Description=LedgerSMB Starman
+Description=LedgerSMB 1.7 Starman
 After=network.target
 
 [Service]
-WorkingDirectory=WORKING_DIR
+WorkingDirectory=/usr/share/ledgersmb-1.7
 
 # In case you installed dependencies into a 'local::lib'
 # make sure you set the PERL5LIB environment variable
 # to the correct location by uncommenting the line below
 #Environment=PERL5LIB=/path/to/local-lib/lib/perl5
+Environment=LSMB_CONFIG_FILE=/etc/ledgersmb-1.7/ledgersmb.conf
 
 # Be sure to set a user and group below
 # which don't have write access to the directories
 # holding the LedgerSMB sources
-User=ledgersmb
-Group=ledgersmb
+User=ledgersmb-1_7
+Group=ledgersmb-1_7
 ExecStart=/usr/bin/starman                     \
     --listen localhost:5762                    \
     -I lib                                     \
